lib/chromic_pdf/supervisor.ex

```diff
@@ -158,18 +158,28 @@ defmodule ChromicPDF.Supervisor do
 
       @type telemetry_metadata_option :: {:telemetry_metadata, map()}
 
+      @type wait_for_option ::
+              {:wait_for,
+               %{
+                 required(:selector) => String.t(),
```

Comment:
tiny last question: `String.t()` and `binary()` are equivalent AFAIK, but `String.t()` has the notion of "should always be UTF-8-encoded" -> are HTML attributes and selectors utf8 encoded?

Reply:
Good question, not that tiny... (be careful what you ask for) 😄  Here I used `String.t()` to tell that this here is a string, not arbitrary binary data.

Attribute names can contain quite a range of characters: https://html.spec.whatwg.org/multipage/syntax.html#attributes-2

The selector API in JS uses strings but I haven't been able to confirm what exactly is allowed.

I added an integration test with ISO-8859 (latin-1) encoded HTML file and it does work with character `ö` in the attribute name and selector in both HTML and Elixir source.

Reply:
hmm, given the specification doesn't say anything about character encoding, I think I'd prefer `binary()` here. Sorry to be so picky 😄